src/main/java/Butler.java

```diff
@@ -0,0 +1,216 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Butler {
+
+    private static void addTask(String input, ArrayList<Task> taskList, Boolean isNew) {
+        String[] inputList = input.split(" ", 2);
+        String reply = "";
+        Task addedTask;
+        String summary;
+        Boolean isValidCommand = false;
+        try {
+            switch (inputList[0]) {
+                case "todo":
+                    try {
```

Comment:
Is the nesting too deep? Consider abstraction to your other classes

Reply:
I am with @raythx98 on this, perhaps you can make use of your Parser and Storage class?

Reply:
I have done the abstraction in my recent commits. Thank you for the comments.